usermanager/src/main/java/org/pesmypetcare/usermanager/clients/pet/PetManagerClient.java

```diff
@@ -0,0 +1,480 @@
+     */
+    public Object getSimpleField(String accessToken, String username, String petName, String field)
+            throws ExecutionException, InterruptedException {
+        PetData.checkSimpleField(field);
+        taskManager = taskManager.resetTaskManager();
+        taskManager.setTaskId(GET);
+        StringBuilder response = taskManager
+                .execute(BASE_URL + PETS_PATH + username + SLASH + petName + "/simple/" + field, accessToken).get();
+        if (response == null) {
+            return null;
+        }
+        String[] split = response.toString().split("\"");
+        if (split.length == 3) {
+            return Double.parseDouble(split[2].split(":")[1].split("}")[0]);
+        } else if (split.length == 1) {
+            return split[0];
+        } else if (split.length == 5) {
+            return split[3];
+        }
+        return null;
+    }
+
+    /**
+     * Updates de gender of a pet.
+     *
+     * @param accessToken The personal access token for the account
+     * @param username The pet's owner username
+     * @param petName The pet's name
+     * @param field The field to update
+     * @param newValue The new field value
+     * @return The response code
+     * @throws ExecutionException When the retrieval fails
+     * @throws InterruptedException When the retrieval is interrupted
+     */
+    public int updateSimpleField(String accessToken, String username, String petName, String field, Object newValue)
+            throws ExecutionException, InterruptedException {
+        PetData.checkSimpleFieldAndValues(field, newValue);
+        taskManager = taskManager.resetTaskManager();
+        Map<String, Object> reqData = new HashMap<>();
+        reqData.put(VALUE_KEY, newValue);
+        taskManager.setTaskId(PUT);
+        taskManager.setReqBody(new JSONObject(reqData));
+        StringBuilder response = taskManager
+                .execute(BASE_URL + PETS_PATH + username + SLASH + petName + "/simple/" + field, accessToken).get();
+        return Integer.parseInt(response.toString());
+    }
+
+
+    /**
+     * Deletes the map for the specified field of the pet on the database.
+     *
+     * @param accessToken The personal access token for the account
+     * @param username The pet's owner username
+     * @param petName The pet's name
+     * @param field Name of the field to delete
+     * @throws ExecutionException When the retrieval fails
+     * @throws InterruptedException When the retrieval is interrupted
+     */
+    public int deleteFieldCollection(String accessToken, String username, String petName, String field)
+            throws ExecutionException, InterruptedException {
+        PetData.checkCollectionField(field);
+        taskManager = taskManager.resetTaskManager();
+        taskManager.setTaskId(DELETE);
+        StringBuilder response = taskManager
+                .execute(BASE_URL + PETS_PATH + username + SLASH + petName + "/collection/" + field, accessToken).get();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/collection/" appears 7 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5561038)